sdk/test-utils/recorder-new/MIGRATION.md

```diff
@@ -0,0 +1,160 @@
+# Migration Guide
+
+This document outlines key differences between the legacy recorder and the new Unified Recorder client.
+
+## Installing the Unified Recorder
+
+The first step is to install the unified recorder client library in your package by running
+
+```bash
+$ rush add --dev -p @azure-tools/test-recorder-new
+```
+
+You will use this library to control the recorder from your tests. The API is somewhat similar to the legacy recorder, albeit with some important differences. These will be discussed below.
```

Comment:
```suggestion
You will use this library to control the recorder from your tests. The API is similar to the legacy recorder. Differences will be discussed below.
```

This sentence sounds doubtful. Here’s an idea, but there are other ways to make it sound more secure, of course.